Header header
geometry_msgs/PoseStamped pose
bool avoiding
